src/components/InputGroup.tsx

```diff
@@ -15,11 +15,16 @@ type InputGroupProps = {
 	label: string
 	type?: HTMLInputTypeAttribute
 	numLines?: number
+	alignStart?: boolean
 }
 
 export default function InputGroup(props: InputGroupProps) {
+	const alignItems = props.alignStart != null && props.alignStart == true
+		? ""
+		: "items-center"
+
 	return (
-		<div className={'w-full flex flex-col items-center justify-start'}>
+		<div className={"w-full flex flex-col justify-start " + alignItems}>
```

Comment:
This is fine, but we can simplify this to:

```<div className={"w-full flex flex-col justify-start " + (props.alignItems ?? "items-center")}>```

Then simply passing a string to override, whether that be "items-start" or something else, and deleting the alignItems variable. Same goes for SelectGroup.tsx!

Might also be a good idea to include the justify in here as well so we can position wherever and however in the future.